tests/Harvest/Importers/AuthorityImporterTest.php

```diff
@@ -43,7 +43,7 @@ public function testUpdateRelated() {
         $row = $this->getData();
         $importer = $this->initImporter($row);
 
-        $authority = $importer->import($row, $result = new Result());
+        $authority = $importer->import($row, $result = new Progress());
```

Comment:
Total nit pick, but these are making me uneasy:
```suggestion
        $authority = $importer->import($row, new Progress());
```